test/services/monitoring-stations/fetch-monitoring-stations.service.test.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, before } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const LicenceHelper = require('../../support/helpers/licence.helper.js')
+const MonitoringStationHelper = require('../../support/helpers/gauging-station.helper.js')
+const LicenceMonitoringStationHelper = require('../../support/helpers/licence-gauging-station.helper.js')
+
+// Thing under test
+const FetchMonitoringStationsService = require('../../../app/services/monitoring-stations/fetch-monitoring-stations.service.js')
+
+describe('Fetch Monitoring Stations service', () => {
+  let monitoringStation
+  let monitoringStationId
+  let licence
+  let licenceId
+  let licenceMonitoringStation
+
+  describe('when a monitoring station has linked licences', () => {
+    before(async () => {
+      licence = await LicenceHelper.add()
+      licenceId = licence.id
+
+      monitoringStation = await MonitoringStationHelper.add()
+      monitoringStationId = monitoringStation.id
+
+      licenceMonitoringStation = await LicenceMonitoringStationHelper.add({
+        licenceId,
+        gaugingStationId: monitoringStationId
+      })
+    })
+
+    it('returns the matching monitoring station, and licences linked to the monitoring station', async () => {
```

Comment:
Now that you are ordering in the fetch I would add some unit tests to check that the object is being ordered correctly